xcp_d/interfaces/bids.py

```diff
@@ -129,18 +131,19 @@ def _run_interface(self, runtime):
                 )
             )
 
-            # TODO: Collect from templateflow once it's uploaded.
-            # FreeSurfer: fs_?/fs_?-to-fs_LR_fsaverage.?_LR.spherical_std.164k_fs_?.surf.gii
-            self._results["sphere_to_sphere"] = pkgrf(
-                "xcp_d",
-                (
-                    f"data/standard_mesh_atlases/fs_{hemisphere}/"
-                    f"fs_{hemisphere}-to-fs_LR_fsaverage.{hemisphere}_LR.spherical_std."
-                    f"164k_fs_{hemisphere}.surf.gii"
-                ),
```

Comment:
@madisoth is this `tpl-fsLR_space-fsaverage_hemi-<L|R>_den-164k_sphere.surf.gii` (which is on TemplateFlow)? Or would it be `tpl-fsaverage_space-fsLR_hemi-<L|R>_den-164k_sphere.surf.gii` (which I think we would need to add to TemplateFlow)?

Reply:
Currently not found.